libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -1705,6 +1705,10 @@ const char *AP_Mission::Mission_Command::type() const {
         return "LoitUnlim";
     case MAV_CMD_NAV_LOITER_TIME:
         return "LoitTime";
+    case MAV_CMD_NAV_LOITER_TURNS:
```

Comment:
We have space in the status text for longer messages, could we make these less programmery and spell the names all the way out? There are 32 chars available if the `(ID=%)` format is kept, and the longest string I'm seeing is 23 characters for continue and change altitude. At that point we could format the string to be more like "Executing continue and change alt (waypoint 700)" which I think would read a lot better on the GCS side.

Reply:
Current plan is for me to modify the mavlink generator to also supply text strings for these `MAV_CMD_x` things.  Strings are already available for the messages.

In your example that would be "Executing CONTINUE_AND_CHANGE_ALT (waypoint 700)"

Not as plain-texty  as yours, but the intent of the item pops out :-)

Reply:
Quick skim of the commands the longest one I can see expecting in a mission is `DO_SET_CAM_TRIGG_INTERVAL` which puts us right at 50 characters. I may have missed one though, the longer ones I can find don't really make sense in missions, but sticking with the generated approach does push us closer to string length limits.

Reply:
We're going with custom short-strings now.